app/src/main/java/com/madonasyombua/growwithgoogleteamproject/ui/fragment/ProfileFragment.java

```diff
@@ -116,4 +117,19 @@ public void onDetach() {
         // TODO: Update argument type and name
         void onFragmentInteraction(Uri uri);
     }
+    private void setStatus(boolean online){
+        if(online)
+        {
+            mBinding.status.setCompoundDrawables(getActivity().getResources()
```

Comment:
can you please refactor the  call of the drawable , the différence is the setText value
you can do that if you want
String statusText = online ?  "Online" : "Offline";
mBinding.status.setCompoundDrawables(getActivity().getResources()
.getDrawable(R.drawable.ic_offline),null,null,null);
 mBinding.status.setText(statusText);

Reply:
What is wrong with the way it is now?

Reply:
nothing :) just refactoring, you have duplicated code the only difference is the texte that you will set it.

Reply:
I already have this error Error:Execution failed for task ':app:compileDebugJavaWithJavac'.
> android.databinding.tool.util.LoggedErrorException: Found data binding errors.
  ****/ data binding error ****msg:Cannot find the setter for attribute 'android:text' with parameter type boolean on android.widget.TextView.
  file:C:\Users\madon\AndroidStudioProjects\GrowWithGoogleTeamProject\app\src\main\res\layout\fragment_profile.xml
  loc:75:44 - 75:54
  ****\ data binding error ****

Reply:
It's cleaner that way avoids creating unnecessary variable. Between, this isn't the last commit

Reply:
I think something broke lol

Reply:
Ok, my bad

Reply:
never mind nothing broke. Just check the last commit

Reply:
ah no, just a suggestion :)

Reply:
@Madonahs you have to clean an rebuild the project

Reply:
Okay! I will do that when I get back from work. Everything works good on my side. Just make sure the profile-activity branch is clean and only code from profileActivity is been executed

Reply:
run this commit 226cb7f1cae8eff096b0bc717d9c2d9a66646082 this is the last one